package.json

```diff
@@ -42,7 +42,7 @@
     "test:unit:login": "BABEL_ENV=test mocha --opts src/platform/testing/unit/mocha.opts --recursive 'test/login/**/*.unit.spec.js?(x)' ",
     "test:unit:personalization": "BABEL_ENV=test mocha --opts src/platform/testing/unit/mocha.opts --recursive 'src/applications/personalization/**/tests/**/*.unit.spec.js?(x)' ",
     "test:unit:db": "BABEL_ENV=test mocha --opts src/platform/testing/unit/mocha.opts --recursive 'test/disability-benefits/**/*.unit.spec.js?(x)' ",
-    "test:unit:4142": "BABEL_ENV=test mocha --opts src/platform/testing/unit/mocha.opts --recursive 'src/applications/disability-benefits/4142/**/tests/**/*.unit.spec.js?(x)' ",
+    "test:unit:4142": "BABEL_ENV=test mocha --opts src/platform/testing/unit/mocha.opts --recursive 'src/applications/disability-benefits/526EZ/**/tests/**/*.unit.spec.js?(x)' ",
```

Comment:
Should we change the name of the script to `test:unit:526`, since we have to deal with those tests as well?